### PR TITLE
Add gisqus comments

### PR DESCRIPTION
**Pull request by dodalovic**

### CODE REVIEW

File: src/theme/BlogPostItem/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import BlogPostItem from '@theme-original/BlogPostItem';
+
+export default function BlogPostItemWrapper(props) {
+  return (
+    <>
```
Comment (m19v) [inline src/theme/BlogPostItem/index.js:6]: Here is the Giscus component missing. Please add it as I have shown in blog post. 
```js
import React from 'react';
import { useBlogPost } from '@docusaurus/theme-common/internal'
import BlogPostItem from '@theme-original/BlogPostItem';
import GiscusComponent from '@site/src/components/GiscusComponent';
import useIsBrowser from '@docusaurus/useIsBrowser';

export default function BlogPostItemWrapper(props) {
  const { metadata, isBlogPostPage } = useBlogPost()
  const isBrowser = useIsBrowser();

  const { frontMatter, slug, title } = metadata
  const { enableComments } = frontMatter

  return (
    <>
      <BlogPostItem {...props} />
      {(enableComments && isBlogPostPage) && (
        <GiscusComponent />
      )}
    </>
  );
}
```

Comment (dodalovic) [reply src/theme/BlogPostItem/index.js:6]: After creating that thing in `/src/theme/index.js`, I get bunch of errors:

```
Module not found: Error: Can't resolve '@theme/SearchPage' in '/home/dod/me/freelance/dodalovic.github.io/.docusaurus'
Module not found: Error: Can't resolve '@theme/SearchPage' in '/home/dod/me/freelance/dodalovic.github.io/.docusaurus'
Module not found: Error: Can't resolve '@theme/Root' in '/home/dod/me/freelance/dodalovic.github.io/node_modules/@docusaurus/core/lib/client'
Module not found: Error: Can't resolve '@theme/SiteMetadata' in '/home/dod/me/freelance/dodalovic.github.io/node_modules/@docusaurus/core/lib/client'
Module not found: Error: Can't resolve '@theme/Loading' in '/home/dod/me/freelance/dodalovic.github.io/node_modules/@docusaurus/core/lib/client/exports'
Module not found: Error: Can't resolve '@theme/NotFound' in '/home/dod/me/freelance/dodalovic.github.io/node_modules/@docusaurus/core/lib/client/exports'
Module not found: Error: Can't resolve '@theme/NotFound' in '/home/dod/me/freelance/dodalovic.github.io/node_modules/@docusaurus/core/lib/client/exports'
Module not found: Error: Can't resolve '@theme/Error' in '/home/dod/me/freelance/dodalovic.github.io/node_modules/@docusaurus/core/lib/client/exports'
Module not found: Error: Can't resolve '@theme/prism-include-languages' in '/home/dod/me/freelance/dodalovic.github.io/node_modules/@docusaurus/theme-classic/lib'
Module not found: Error: Can't resolve '@theme/Layout' in '/home/dod/me/freelance/dodalovic.github.io/src/pages'
client (webpack 5.76.3) compiled with 38 errors
Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/home/dod/me/freelance/dodalovic.github.io/src/theme/index.js'

```

Comment (dodalovic) [reply src/theme/BlogPostItem/index.js]: Seems I've fixed it 🤔

Comment (m19v) [reply src/theme/BlogPostItem/index.js]: Please do not create it under `src\theme` but just edit already existing component in `src\theme\BlogPostItem` index.js.

Comment (m19v) [reply src/theme/BlogPostItem/index.js]: You are welcome!

**Pull request merged**